pygeos/coordinates.py

```diff
@@ -101,6 +101,10 @@ def get_coordinates(geometry, include_z=False):
     include_z : bool, default False
         Whether to include the third dimension in the output. If True, and a
         geometry has no third dimension, the z-coordinates will be NaN.
+    return_index : bool, default False
+        Whether to return the index of each returned geometry as a separate
```

Comment:
```suggestion
        If True will return the index of each returned geometry as a separate
```

Reply:
What is wrong with the "whether to"? we used it also in the `include_z`

Reply:
Either is fine, I just thought "If True" was a bit more explicit.  Could be updated for `include_z` too.  😄 

Elsewhere we seem to generally use "If True..." or a somewhat more verbose variant ("If <parameter> is set to True")

Reply:
Ok, I normalized these docstrings to "If True, ..."